balancer-js/src/modules/pools/pool-types/concerns/composableStable/exit.concern.integration.spec.ts

```diff
@@ -0,0 +1,182 @@
+// yarn test:only ./src/modules/pools/pool-types/concerns/composableStable/exit.concern.integration.spec.ts
```

Comment:
Can you add a test for token ordering, similar to `should build the same for reversed array order for tokens and amounts` in join.